xlators/mgmt/glusterd/src/glusterd-pmap.c

```diff
@@ -260,6 +261,7 @@ port_brick_bind(xlator_t *this, int port, char *brickname, void *xprt,
         if (ret > 0) {
             ret = 0;
             tmp_port->brickname = gf_strdup(new_brickname);
+            GF_FREE(new_brickname);
```

Comment:
It doesn't make sense to duplicate `new_brickname` and just after that destroy it. Why you don't simply do:

```suggestion
            tmp_port->brickname = new_brickname;
```

Reply:
It can be done that way, but I was confused if that's a good practice. But if this method is fine, then I will update the code.